receiver/postgresqlreceiver/go.mod

```diff
@@ -4,31 +4,32 @@ go 1.17
 
 require (
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/collector v0.38.1-0.20211103215828-cffbecb2ac9e
+	go.opentelemetry.io/collector v0.39.0
 	go.uber.org/multierr v1.7.0
 )
 
+require go.opentelemetry.io/collector/model v0.39.0
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
It's probably ok to leave, but it wasn't there before this PR. Component is partially added and not using this yet.